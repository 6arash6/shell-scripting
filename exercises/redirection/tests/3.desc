this should check the output of first-engineer.sh
