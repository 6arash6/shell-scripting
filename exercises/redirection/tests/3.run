./last-5-records.sh ./data/yellow-pages.txt
