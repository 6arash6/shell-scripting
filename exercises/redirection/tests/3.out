scientist
Emma Nelson
3694 Maple Avenue
Tel Aviv, Israel

doctor - ophthalmologist
Dr. David Lewis
7185 Cedar Lane
Rome, Italy

accountant
Mia Lee
8389 Oak Lane
Fort Worth, TX, USA

pilot
Tina Martin
6044 Cedar Blvd.
Tel Aviv, Israel

paramedic
Wendy Jackson
4183 Oak Lane
Bangkok, Thailand


